app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
```

Comment:
Code Style: Use `( )` while importing multiple classes/modules. You've done it correctly, but it's worth mentioning that this is a good practice according to the checklist.